### DISCUSSION TITLE
Is there a Otterscan similar on Reth?

### DISCUSSION THREAD

**Discussion by TangMonk** [paradigmxyz/reth#7951] [General]
Just like Otterscan on Erigon

**Comment by onbjerg** [comment]
No, but there is a tracking issue for Otterscan support: https://github.com/paradigmxyz/reth/issues/3726

**Comment by TangMonk** [comment]
> No, but there is a tracking issue for Otterscan support: #3726*

Seems that #3726 has implemented a lot of Otterscan API, so will it be completed in the future?

**Reply by onbjerg** [reply]
The API itself is completed, but they are just returning dummy responses. It will be completed in the future but it is not a priority right now
